desktop/core/src/desktop/static/desktop/locales/en/translation.json

```diff
@@ -138,7 +138,6 @@
   "Drop a SQL file here": "Drop a SQL file here",
   "Drop iPython/Zeppelin notebooks here": "Drop iPython/Zeppelin notebooks here",
   "Edit": "Edit",
-  "Edit list...": "Edit list...",
```

Comment:
@JohanAhlen @bjornalm - Does it makes sense to remove it from here explicitly or we should regenerate this translation json again? If its the latter, then how can we generate it (maybe as a separate commit)?

Reply:
Yeah, good to remove here if it's no longer used.